src/Datadog.Trace.ClrProfiler.Managed/Integrations/WebRequestIntegration.cs

```diff
@@ -25,12 +25,12 @@ public static object GetResponse(object webRequest)
         {
             var request = (WebRequest)webRequest;
 
-            if (!IsTracingEnabled(request))
+            if (!(request is HttpWebRequest) || !IsTracingEnabled(request))
```

Comment:
Short circuits when it's something like a WebPackRequest, which we should instrument and test specifically.